client/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I'm guessing this is the default `.gitignore` file generated upon creating the react app. Because we are here in a multi-project repository, you'll have to merge the contents of this new file into the existing `.gitignore` that's at the root level. Also, you'll have to prefix most of the paths with `client/` because the paths will be relative from the root.